README.md

```diff
@@ -1,24 +1,37 @@
-# Ukrmap Rails [![Build Status](https://travis-ci.org/ukrmap/ukrmap_rails.svg?branch=master)](https://travis-ci.org/ukrmap/ukrmap_rails)
+# Ukrmap
+  Rube's version 2.3.1
+  Rail's version 5.0.1
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Install Instructions
 
-Things you may want to cover:
+### Installation
 
-* Ruby version
+1. Clone repository:
 
-* System dependencies
+    $git clone https://github.com/ukrmap/ukrmap_rails.git
 
-* Configuration
+2. Change directory:
 
-* Database creation
+    $cd ukrmaps_rails
```

Comment:
Please add correct code formatting - 4 spaces.
Also add a space after $.

Another option format code with starting ` ``sh   and ending   ``` to add code highlighting.
See https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet